### DISCUSSION TITLE
When I should use dependencies in Provider's constructor?

### DISCUSSION THREAD

**Discussion by vimhead** [rrousselGit/riverpod#977] [Q&A]
`Provider((ref) => Repo(), dependencies: [])`

**Comment by rrousselGit** [comment]
Specify `dependencies` when it depends on a provider that you may "scope".

If your provider isn't depending on anything, there's no value in specifying `dependencies`

**Reply by vimhead** [reply]
Could you please provide an example? Haven't found it in the docs.
